commons/src/main/java/com/simpletools/calculator/commons/helpers/CalculatorImpl.kt

```diff
@@ -150,4 +151,8 @@ class CalculatorImpl(calculator: Calculator) {
         firstNumber *= -1
         setValue(Formatter.doubleToString(firstNumber))
     }
+
+    private fun formatUserInputTrailingZero(number: Double): String{
```

Comment:
Why don't we include this in the formatter

Reply:
Agreed, this function would be appropriately encapsulated in the Formatter with a name like doubleFixedPointToString.

Reply:
Fixed plus I added test for every formatter method